packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -514,4 +502,29 @@ export class TextArea
       footerWidth,
     };
   }
+
+  private replacePlaceHoldersInMessages(): string {
```

Comment:
Nitpick: was the move of this function intentional?

Reply:
yup, this is intentional to move all the private method to the end.